### PR TITLE
wording changes

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: technology.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ canonical: 'https://druid.apache.org/technology'
 ---
 
 Apache Druid is an open source distributed data store.
-Druid’s core design combines ideas from [data warehouses](https://en.wikipedia.org/wiki/Data_warehouse), [timeseries databases](https://en.wikipedia.org/wiki/Time_series_database), and [search systems](https://en.wikipedia.org/wiki/Full-text_search) to create a unified system for real-time analytics for a broad range of [use cases](/use-cases). Druid merges key characteristics of each of the 3 systems into its ingestion layer, storage format, querying layer, and core architecture.
+Druid’s core design combines ideas from [data warehouses](https://en.wikipedia.org/wiki/Data_warehouse), [timeseries databases](https://en.wikipedia.org/wiki/Time_series_database), and [search systems](https://en.wikipedia.org/wiki/Full-text_search) to create a powerful query serving layer for a broad range of [use cases](/use-cases). Druid merges key characteristics of each of the 3 systems into its ingestion layer, storage format, querying layer, and core architecture.
```
Comment (gianm) [inline technology.md:9]: IMO change this to:

> to create a powerful, high performance analytics database

It's a "what Druid is" sentence, and that is a "high performance real-time analytics database".

File: use-cases.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ sectionid: use-cases
 canonical: 'https://druid.apache.org/use-cases'
 ---
 
-## Real-time analytics and intelligence
+## Power data applications
```
Comment (gianm) [inline use-cases.md:8]: This headline is too specific for the content below (many of the use cases are data applications, but not all of them are).

Suggest rewording it to something like:

> Power real-time analytics, data applications, and more

File: package.json [hunk 3]
```diff
@@ -21,6 +21,6 @@
     "gulp": "3.9.1",
     "gulp-sass": "4.0.1",
     "natives": "^1.1.6",
-    "node-sass": "4.9.2"
+    "node-sass": "^4.14.1"
```
[the remark this answers is not in the document]
Comment (fjy) [reply package.json:24]: no - this should be in gitignore. I'll revert.

File: .gitignore [hunk 4]
```diff
@@ -3,6 +3,8 @@ Gemfile.lock
 .DS_store
 .idea
 .jekyll-metadata
+package.json
```
Comment (gianm) [inline .gitignore:6]: @vogievetsky can you please comment on whether these belong in .gitignore or not?

**Review by gianm — COMMENTED** [review]
Content LGTM but I want to double check on the package machinery

File: .gitignore (hunk 4 above)
Comment (vogievetsky) [reply .gitignore:6]: definitely not!

**Comment by vogievetsky** [comment]
Please revert all changes to package.json and package-lock.json and (.gitignore) and it is g2g
